client/main.lua

```diff
@@ -377,6 +383,64 @@ RegisterNUICallback("incidentSearchPerson", function(data, cb)
     cb(true)
 end)
 
+-- Handle sending the player to jail
+-- Uses QB-Core/qb-policejob function to send the player to jail
+-- If you use a different jail system, you will need to change this
+RegisterNUICallback("sendToJail", function(data, cb)
+    local citizenId, sentence = data.citizenId, data.sentence
+
+    -- Gets the player id from the citizenId
+    local p = promise.new()
+    QBCore.Functions.TriggerCallback('mdt:server:GetPlayerSourceId', function(result)
+        p:resolve(result)
+    end, citizenId)
+
+    local targetSourceId = Citizen.Await(p)
+
+    if sentence > 0 then
+        -- Uses qb-policejob JailPlayer event
+        TriggerServerEvent("police:server:JailPlayer", targetSourceId, sentence)

```

Comment:
I figured the majority of consumers are probably using qbcore + qb-policejob so most will be able to use this out of the box w/o needing to change it